python/plugins/processing/algs/gdal/gdal2xyz.py

```diff
@@ -49,6 +53,16 @@ def initAlgorithm(self, config=None):
                                                      self.tr('Band number'),
                                                      1,
                                                      parentLayerParameterName=self.INPUT))
+
+        self.addParameter(QgsProcessingParameterNumber(self.SRCNODATA,
+                                                       self.tr('Use the given value as nodata'),
```

Comment:
Let's match these strings to the equivalents used by other algorithms:

```suggestion
                                                       self.tr('Input pixel value to treat as "nodata"'),
```